### DISCUSSION TITLE
Is it possible to take players ping from api

### DISCUSSION THREAD

**Discussion by limpeex** [discord-jda/JDA#2263] [General]
Is it possible to take players ping from api

**Comment by HardstylesDev** [comment]
You can get the gateway ping of the bot, but IFAIK it's not possible to get the ping of a user, assuming that's what you meant by player

**Comment by PearlMiller** [comment]
Yes, it's possible to get a player's ping from an API. In simple terms, an API is like a waiter in a restaurant. You (the client) can ask the waiter (API) to bring you specific information (like a player's ping) from the kitchen (server). The server is where all the data is stored, including information about players and their pings.

Imagine the player's ping as a special dish in the kitchen. You can ask the waiter (API) to check if that dish (player's ping) is ready, and the waiter will bring you the information back. In the world of online gaming, this is very useful because players and game developers can monitor internet connections and ensure everyone is having a smooth gaming experience.

**Reply by JellyBrick** [reply]
Generated by LLM? 😅
